### ISSUE TITLE
Application crashes on interaction with push notification

### ISSUE THREAD

**Issue by ndizazzo** [ndizazzo/android-parse-titanium-module#20]
### Issue Summary

When you interact with a push notification by tapping it in the notification list, the host application crashes on resume.

#### Steps to Reproduce

* Send a push notification using the Parse.com control panel
* Wait for the notification to be received on the device
* Tap the notification in the notification list

#### Logcat Output

```
E/AndroidRuntime(16101): FATAL EXCEPTION: main
E/AndroidRuntime(16101): Process: com.xxx.yyy, PID: 16101
E/AndroidRuntime(16101): java.lang.RuntimeException: Unable to start receiver com.parse.ParsePushBroadcastReceiver: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat= flg=0x1000c000 (has extras) }
E/AndroidRuntime(16101): 	at android.app.ActivityThread.handleReceiver(ActivityThread.java:2596)
E/AndroidRuntime(16101): 	at android.app.ActivityThread.access$1800(ActivityThread.java:169)
E/AndroidRuntime(16101): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1356)
E/AndroidRuntime(16101): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(16101): 	at android.os.Looper.loop(Looper.java:146)
E/AndroidRuntime(16101): 	at android.app.ActivityThread.main(ActivityThread.java:5487)
E/AndroidRuntime(16101): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(16101): 	at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime(16101): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
E/AndroidRuntime(16101): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
E/AndroidRuntime(16101): 	at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime(16101): Caused by: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat= flg=0x1000c000 (has extras) }
E/AndroidRuntime(16101): 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1660)
E/AndroidRuntime(16101): 	at android.app.Instrumentation.execStartActivitiesAsUser(Instrumentation.java:1487)
E/AndroidRuntime(16101): 	at android.app.ContextImpl.startActivitiesAsUser(ContextImpl.java:1461)
E/AndroidRuntime(16101): 	at android.content.ContextWrapper.startActivitiesAsUser(ContextWrapper.java:356)
E/AndroidRuntime(16101): 	at android.content.ContextWrapper.startActivitiesAsUser(ContextWrapper.java:356)
E/AndroidRuntime(16101): 	at android.app.TaskStackBuilder.startActivities(TaskStackBuilder.java:221)
E/AndroidRuntime(16101): 	at android.app.TaskStackBuilder.startActivities(TaskStackBuilder.java:232)
E/AndroidRuntime(16101): 	at android.app.TaskStackBuilder.startActivities(TaskStackBuilder.java:208)
E/AndroidRuntime(16101): 	at com.parse.TaskStackBuilderHelper.startActivities(TaskStackBuilderHelper.java:19)
E/AndroidRuntime(16101): 	at com.parse.ParsePushBroadcastReceiver.onPushOpen(ParsePushBroadcastReceiver.java:202)
E/AndroidRuntime(16101): 	at com.parse.ParsePushBroadcastReceiver.onReceive(ParsePushBroadcastReceiver.java:108)
E/AndroidRuntime(16101): 	at android.app.ActivityThread.handleReceiver(ActivityThread.java:2581)
E/AndroidRuntime(16101): 	... 10 more
```

**Comment by ndizazzo** [comment]
Referencing this issue on SO:

http://stackoverflow.com/questions/26154855/exception-when-opening-parse-push-notification

**Comment by ndizazzo** [comment]
Resolved by commit 1ad63922bf0bdb693fb0cc3a783a02dc03e934d6
